R/modeltests.R

```diff
@@ -43,24 +43,26 @@ modeltests <- function(
   setwd(mydir)
   
   message("\n=========================================================
-  Begin of AMT procedure ", format(Sys.Date(), "%Y-%m-%d"), " in ", mydir, 
+  Begin of AMT procedure ", format(Sys.time(), "%d-%m-%Y %H:%M:%S"), " in ", mydir, 
```

Comment:
I'd recommend to stick with the usual year-month-day scheme to avoid confusion, or is there a specific reason not to here?
```suggestion
  Begin of AMT procedure ", format(Sys.time(), "%Y-%m-%d %H:%M:%S"), " in ", mydir, 
```